python/py-pyside2/Portfile

```diff
@@ -16,19 +16,25 @@ homepage                https://wiki.qt.io/Qt_for_Python
 license                 {GPL-2 GPL-3+ LGPL-3}
 master_sites            https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${version}-src
 distname                pyside-setup-opensource-src-${version}
-use_xz                  yes
 
-checksums               rmd160  09c0517e0fe07722e799a32c0248cd7bf8b3a581 \
-                        sha256  b306504b0b8037079a8eab772ee774b9e877a2d84bab2dbefbe4fa6f83941418 \
-                        size    3472624
+checksums               rmd160  6d57490a3adf53d659b6192aabd7c65ff7090915 \
+                        sha256  91e78e4e3b31ebb0053c2e4f1029703e578615695450c0dd03072248d713b391 \
+                        size    3473548
 
-python.versions         27 36 37 38 39
+python.versions         36 37 38 39 310
 
 set llvm_version        13
 
 if {${name} ne ${subport}} {
     PortGroup           qt5 1.0
 
+    # fix error related to _Py_Mangle, which was half-patched in this version
+    # see: https://bugzilla.redhat.com/show_bug.cgi?id=1990768#c9
+    patchfiles-append   patch-py-mangle.diff
+
+    # fix error with shiboken2 doc building
+    patchfiles-append   patch-shiboken2-docs.diff
+
```

Comment:
perhaps these patches are not needed anymore?

Reply:
Nope, build still fails when either patch is removed